tensorbay/opendataset/AnimalPose/loader.py

```diff
@@ -4,6 +4,7 @@
 #
 # pylint: disable=invalid-name
 # pylint: disable=missing-module-docstring
+"""Dataloader of AnimalPose5 dataset and AnimalPose7 dataset."""
```

Comment:
Dataloaders, other docstrings are also need to change

Reply:
whether there are more than two datasets I need to change to the dataloaders or I have to change all the dataloader to loaders

Reply:
use "Dataloaders" when there are two datasets